_layouts/cask.html

```diff
@@ -6,6 +6,13 @@
 {%- assign formula_path = "formula" -%}
 {%- assign c = site.data.cask[token] -%}
 <h2>{{ token }}</h2>
+
+{%- capture code -%}
+brew install cask {{ token }}
```

Comment:
```suggestion
brew install {{ token }}
```

Reply:
Should be `brew cask install {{ token }}` right?

![image](https://user-images.githubusercontent.com/10703445/93849777-71240200-fcca-11ea-9785-0f4110155095.png)

Reply:
You're right. I got confused because `upgrade` and `reinstall` now also work for casks, `install` doesn't yet.